hles/transformation/src/main/scala/org/broadinstitute/monster/dap/OwnerTransformations.scala

```diff
@@ -22,8 +22,6 @@ object OwnerTransformations {
 
   /** Parse all owner-related fields out of a raw RedCap record. */
   def mapOwner(rawRecord: RawRecord): HlesOwner = {
-    val education = rawRecord.getOptionalNumber("od_education")
```

Comment:
Didn't need this variable anymore since we aren't processing it to pull out an "other" field anymore.